e2e/transform/multiple-transformers/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "jest": {
     "transform": {
-      "^.+\\.css$": "<rootDir>/CssPreprocessor.js",
-      "^.+\\.js$": "<rootDir>/JsPreprocessor.js",
-      "^.+\\.svg$": "<rootDir>/FilePreprocessor.js"
+      "^.+\\.css$": "<rootDir>/cssPreprocessor.js",
+      "^.+\\.js$": "<rootDir>/jsPreprocessor.js",
+      "^.+\\.svg$": "<rootDir>/filePreprocessor.js"
```

Comment:
these files are added but the old ones are not removed, can you fix that?

Reply:
They should be removed now. :)